tests/h/auth/policy_test.py

```diff
@@ -31,6 +31,19 @@
     '/api/token',
 )
 
+AUTH_CLIENT_PATHS = (
+    '/api/groups',
+    '/api/groups/',
+)
+
+NON_AUTH_CLIENT_PATHS = (
+    '/api/groups//',
+    '/api/groups/33333',
+    '/api/badge',
+    '/api',
+    '/groups/'
+)
```

Comment:
Could these constants be moved into the `TestAPIAuthenticationPolicy` that they're used in? Just to reduce the amount of travel needed to understand the code. Although if `request.matched_route` is used the code'll be doing simple `==` against a single possible matching string value, instead of any kind of pattern match, so testing with multiple correct and incorrect strings won't be necessary anymore anyway.